include/CppUTest/CppUTestConfig.h

```diff
@@ -32,6 +32,12 @@
 #include "CppUTestGeneratedConfig.h"
 #endif
 
+#ifdef _cpputest_pid_t
+#define PID_T _cpputest_pid_t
```

Comment:
Isn't a typedef a better choice here? `#define` can be harmful at many places.

Reply:
Changed #define to typedef.